src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -354,6 +361,9 @@ private static void HookupHistory(ModelBuilder modelBuilder)
         modelBuilder.Entity<EntityStateHistory>()
             .HasIndex(e => new { e.EntityId, e.FromStateVersion });
 
+        modelBuilder.Entity<ValidatorStateHistory>()
+            .HasIndex(e => new { e.EntityId, e.FromStateVersion });
```

Comment:
When given table is meant to hold data of a specific entity type we usually use more verbose names:

```
ValidationStateHistory => ValidatorEntityId
KeyValueStoreSchemaHistory => KeyValueStoreEntityId
...
```